source/blog/2019-01-06-ovirt-openshift-part-1.html.md

```diff
@@ -0,0 +1,148 @@
+---
+title: oVirt and OKD
+author: rgolan
+tags: oVirt, oVirt 4.2, open source virtualization,kubernetes, openshift, external-storage, flexdriver, provisioner
+date: 2019-01-06 09:01:00 UTC
+---
+
+<img src="/images/blog/2019-01-06/boxhead.png" width="400px"/>
+
+
+
+This is a series of posts to demonstrate how to  install  OKD 3.11 on oVirt and what you can do with it.
+
+**Part I**   -  How to intall OKD 3.11 on oVirt
+
+
+
+# How to intall OKD 3.11 on ovirt (4.2 and up)
+
+Installing OKD on oVirt has many advantages, and its also gotten a lot easier these days. Admins and users who want to take container platform management for a spin, on oVirt, will be encouraged by this.
+
+The installation uses [openshift-ansible](https://github.com/openshift/openshift-ansible) and, specifically the `openshift_ovirt` ansible-role. The integration between OpenShift and oVirt is tighter, and provides storage integration. If you need persistent volumes for your containers you can get that directly from oVirt using **ovirt-volume-provisioner** and **ovirt-flexvolume-driver**.
+
+For the sake of simplicity, this installation will be an all-in-one OpenShift cluster, on a single VM.  
```

Comment:
For the sake of simplicity, this example will cover an all-in-one OpenShift cluster, on a single VM.

Reply:
done